addons/arsenal/functions/fnc_fillSort.sqf

```diff
@@ -0,0 +1,74 @@
+#include "script_component.hpp"
+#include "..\defines.hpp"
+
+params ["_display", "_control", "_sortCtrl"];
+
+lbClear _sortCtrl;
+
+private _sorts = if (ctrlIDC _sortCtrl == 17 && {GVAR(currentLeftPanel) in [IDC_buttonUniform ,IDC_buttonVest, IDC_buttonBackpack]}) then {
```

Comment:
Magic number. Fix spacing around commas.

Reply:
This magic number exists all around `arsenal`, it should be fixed everywhere else too